ntc_templates/templates/alcatel_aos_show_vlan_port.textfsm

```diff
@@ -0,0 +1,12 @@
+Value VLAN_ID (\d+)
+Value PORT ((\d+\/)?\d+\/\d+)
+Value TYPE (default|qtagged)
+Value STATUS (forwarding|inactive)
+
+Start
+  ^\s*vlan\s*port\s*type\s*status\s*$$
+  ^\s*--------+------------+------------+--------------*$$
+  ^\s*------+-------+---------+-------------*$$
```

Comment:
```suggestion
  ^\s*[-+]+\s*$$
```

Condense two separator lines into one.
Added accommodation for trailing white space with `\s*` just in case there's ever a trailing space or two.

If you support my suggestion, click the button here to accept my suggestion. Thank you. :grinning:

Reply:
I love it! Way cleaner and better :)